game/scripts/vscripts/GameMode.ts

```diff
@@ -204,18 +196,37 @@ export class GameMode {
         if (!unit) return;
 
         if (unit.IsBaseNPC()) {
+
+            // Check if this is the real player's hero that just spawned, assign it to the gamemode entity if it is
+            if (unit.IsRealHero()) {
+                unit.IsTempestDouble()
```

Comment:
huh?

Reply:
Was a remnant of something I tried to do. Removed in latest commit.